src/main/java/uk/org/okapibarcode/output/SvgRenderer.java

```diff
@@ -120,13 +139,38 @@ public void render(Symbol symbol) throws IOException {
                 writer.append("   \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n");
             }
 
+            // render rotation clockwise
+            int rotateHeight = height;
+            int rotateWidth = width;
+            String transform;
+            switch (rotation) {
+                case 90:
+                    rotateHeight = width;
+                    rotateWidth = height;
+                    transform = " transform=\"rotate(-270) translate(0,-" + rotateWidth + ")\"";
+                    break;
+                case 180:
+                    transform = " transform=\"rotate(" + rotation + "," + (width/2) + "," + (height/2) + ")\"";
+                    break;
+                case 270:
+                    rotateHeight = width;
+                    rotateWidth = height;
+                    transform = " transform=\"rotate(-90) translate(-"+rotateHeight+",0)\"";
```

Comment:
Can this be `rotate(270)`, or does that cause issues?

Reply:
Same here it doesn't work with +270 but i don't know WHY.

Reply:
Very strange, I just tweaked the result file manually, and 270 looked exactly like -90... unless I did something wrong?!? Can you double check?